sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/samples/README.md

```diff
@@ -0,0 +1,26 @@
+# Azure Event Hubs Checkpoint Store client library samples for Java
+
+Azure Event Hubs Checkpoint Store samples are a set of self-contained Java programs that demonstrate interacting 
+with Azure Event Hubs Checkpoint Store using the client library. 
+
+## Key concepts
+Key concepts are explained in detail [here][sdk_readme_key_concepts].
+
+## Getting started
+Please refer to the [Getting Started][sdk_readme_getting_started] section.
+
+## Samples
+
+- [Create an instance of Storage Container client][sample_container_client]
+- [Consume events from all Event Hub partitions][sample_event_processor]
+
+## Contributing
+
+If you would like to become an active contributor to this project please refer to our [Contribution
+Guidelines](../../CONTRIBUTING.md) for more information.
+
+<!-- Links -->
+[sdk_readme_key_concepts]: ../../README.md#key-concepts
+[sdk_readme_getting_started]: ../../README.md#getting-started
+[sample_container_client]: ./java/com/azure/messaging/eventhubs/checkpointstore/blob/BlobCheckpointStoreSample.java
+[sample_event_processor]: ./java/com/azure/messaging/eventhubs/checkpointstore/blob/EventProcessorBlobCheckpointStoreSample.java
```

Comment:
Do we need one of those `![impressions]` here too?

Reply:
Good catch! Added.